net-im/qtox/qtox-1.7.0-r1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit qmake-utils toolchain-funcs
+
+DESCRIPTION="Most feature-rich GUI for net-libs/tox using Qt5"
+HOMEPAGE="https://github.com/qTox/qTox"
+SRC_URI="https://github.com/qTox/qTox/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk X"
+
+# needed, since tarball provided by github extracts to `qTox`
+S="${WORKDIR}/qTox-${PV}"
+
+RDEPEND="
+	dev-db/sqlcipher
+	dev-libs/libsodium
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5=
```

Comment:
I put the = on dev-qt/qtcore just to trigger rebuild when qt slot is changed. I'll retest today to see what component will show in error like this `qtox: /usr/lib64/libQt5Gui.so.5: version `Qt_5.7' not found (required by qtox)` and update the commit with = on correct dependency.